db/migrate/20220722215911_change_text_profile_column_name.rb

```diff
@@ -0,0 +1,19 @@
+class ChangeTextProfileColumnName < ActiveRecord::Migration[7.0]
+  def change
+    add_column :partner_profiles, :client_capacity, :string
+    add_column :partner_profiles, :primary_contact_name, :string
+    add_column :partner_profiles, :primary_contact_phone, :string
+    add_column :partner_profiles, :primary_contact_mobile, :string
+    add_column :partner_profiles,  :primary_contact_email, :string
+
+    ::Partners::Partner.all.find_each do |partner_profile|
```

Comment:
I generally try to avoid adding models inside migrations since this would cause the migration into fail if `Partners::Partner` either was renamed or removed. To avoid this. I would add
```
class Partners::Partner 
   # add associations you used below.
end
```
This allows the migration to be complete because the definition of the model is here and self-contained.

Reply:
Make sense, good observation. 
Is there a possibility to run some rake task after deploy?

I ask about this, because, probably this code (copy old data to new columns) is better stay in rake task.
Its necessary run only once ( after deploy in production )

Reply:
No, it's fine to make data changes in a migration. Migrations also only run once. :)

Reply:
Nice point @dorner , so what do you suggest to change?
In this case, I believe that PR is ok, do you agree ? @dorner @edwinthinks

Reply:
The only thing I'd say is that rather than defining the class here, I'd do something a bit more meta:

```ruby
klass = Class.new(Partners::Base) do
   self.table_name = 'partner_profiles'
end

klass.all.find_each do |profile|
...
```

The point is that you don't need most of the functionality of the class - all you need is a way to interact with a particular database table. `Partners::Partner` as a constant actually gives the wrong table name by default, so adding it to the migration won't do you any favors.

Reply:
Good point! @dorner